src/main/scala/wdl4s/formatter/SyntaxFormatter.scala

```diff
@@ -98,6 +99,29 @@ class SyntaxFormatter(highlighter: SyntaxHighlighter = NullSyntaxHighlighter) {
     header
   }
 
+  private def formatRuntimeSection(task: Task, level: Int): String = {
+    task.runtimeAttributes.attrs match {
+      case m: Map[String, Seq[String]] if m.nonEmpty =>
```

Comment:
What else could this possibly match to? Isn't task.runtimeAttributes.attrs a Map[String, Seq[String]]?